### DISCUSSION TITLE
Should we wrap comments?

### DISCUSSION THREAD

**Discussion by Alizter** [HoTT/HoTT#1719] [Ideas]
Currently in our style guide we don't wrap comments on purpose. The reason for this is that when viewing the code in an editor like PG and the buffer causes lines to wrap, then it becomes really distracting to read.

On the other hand, reading unwrapped comments on Github is difficult and throughout OCaml and Coq it is conventional to wrap comments. Since we try to keep Coq code within the 80 col margin, perhaps it might be worth doing the same for comments.

I personally find it easier to read wrapped comments, but I am wondering what others think?

**Comment by jdchristensen** [comment]
I would prefer to have comments with manual line breaks in them.  Not only would that be better on github, I'd like it better in emacs as well.  I indicated this with a thumbs-up above, and maybe others can use thumbs-up and thumbs-down to indicate their preferences.

**Comment by mikeshulman** [comment]
FWIW there is a [userstyle](https://github.com/StylishThemes/GitHub-code-wrap) that will wrap long lines when viewing code in GitHub.
